packages/gatsby/src/utils/feedback.ts

```diff
@@ -97,6 +97,7 @@ export async function userPassesFeedbackRequestHeuristic(): Promise<boolean> {
     }
   }
 
+  // 4.b
```

Comment:
remnant of debugging? or it makes to have it?

Reply:
This was intentional. I think this was missing.